packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceArea/ReferenceArea.stories.svelte

```diff
@@ -305,13 +298,79 @@
 		query
 	)}
 	<LineChart x="x" y="y" {data}>
-		<QueryLoad data={referenceAreaData}>
```

Comment:
Query load already exists in annotation components, causing skeletons to render unnecessarily in stories